packages/cojson-transport-ws/src/index.ts

```diff
@@ -43,26 +40,27 @@ export function createWebSocketPeer({
 
     let pingTimeout: ReturnType<typeof setTimeout> | null = null;
 
+    let supportsBatching = batchingByDefault;
+
     websocket.addEventListener("message", function handleIncomingMsg(event) {
-        const msg = JSON.parse(event.data as string);
-        pingTimeout && clearTimeout(pingTimeout);
-
-        if (msg?.type === "ping") {
-            const ping = msg as PingMsg;
-            g.jazzPings ||= [];
-            g.jazzPings.push({
-                received: Date.now(),
-                sent: ping.time,
-                dc: ping.dc,
-            });
-        } else {
-            incoming
-                .push(msg)
-                .catch((e) =>
-                    console.error("Error while pushing incoming msg", e),
-                );
+        let messagesChunk: PingMsg[] | SyncMessage[];
+
+        try {
+            messagesChunk = (event.data as string)
+                .split("\n")
+                .map((msg) => JSON.parse(msg));
+        } catch (e) {
+            console.error("Error while parsing incoming message", e);
+            return;
+        }
```

Comment:
Sometimes the server reiceves a `undefined:1` message.

Not sure where this comes, but without this try/catch it causes the server to restart.

Reply:
interesting, probably a broken client? Or you receive it even in local testing?

Reply:
Yes, both in local and in the production demo.